jsonschema-validation.md

```diff
@@ -320,115 +309,57 @@ can be used alongside the `type` keyword with a value of "integer", or could be
 explicitly defined to always pass if the number is not an integer, which
 produces essentially the same behavior as only applying to integers.
 
-The current IRI for this vocabulary, known as the Format-Annotation vocabulary,
-is: `https://json-schema.org/draft/next/vocab/format-annotation`. The current
-IRI for the corresponding meta-schema is:
-`https://json-schema.org/draft/next/meta/format-annotation`. Implementing
-support for this vocabulary is REQUIRED.
-
-In addition to the Format-Annotation vocabulary, a secondary vocabulary is
-available for custom meta-schemas that defines `format` as an assertion. The IRI
-for the Format-Assertion vocabulary, is:
-`https://json-schema.org/draft/next/vocab/format-assertion`. The current IRI for
-the corresponding meta-schema is:
-`https://json-schema.org/draft/next/meta/format-assertion`. Implementing support
-for the Format-Assertion vocabulary is OPTIONAL.
-
-Specifying both the Format-Annotation and the Format-Assertion vocabularies is
-functionally equivalent to specifying only the Format-Assertion vocabulary since
-its requirements are a superset of the Format-Annotation vocabulary.
-
-### Implementation Requirements
-
-The `format` keyword functions as defined by the vocabulary which is referenced.
-
-#### Format-Annotation Vocabulary
-
-The value of format MUST be collected as an annotation, if the implementation
-supports annotation collection. This enables application-level validation when
-schema validation is unavailable or inadequate.
-
-Implementations MAY still treat `format` as an assertion in addition to an
-annotation and attempt to validate the value's conformance to the specified
-semantics. The implementation MUST provide options to enable and disable such
-evaluation and MUST be disabled by default. Implementations SHOULD document
-their level of support for such validation.[^2]
+Implementing support for `format` as an annotation is REQUIRED (if the
+implementation supports annotation collection).
 
-[^2]: Specifying the Format-Annotation vocabulary and enabling validation in an
-implementation should not be viewed as being equivalent to specifying the
-Format-Assertion vocabulary since implementations are not required to provide
-full validation support when the Format-Assertion vocabulary is not specified.
-
-When the implementation is configured for assertion behavior, it:
+Implementing support for `format` as an assertion is OPTIONAL. Implementations
+which choose to support assertion behavior:
 
+- MUST still collect the keyword's value as an annotation (if the implementation
+  supports annotation collection),
+- MUST provide a configuration option to enable assertion behavior, defaulting to
```

Comment:
No.  In order to ensure interoperability, we need all implementations to have the same out-of-the-box behavior.  Implementations can do whatever, but they need non-spec behavior hidden behind configuration so that the user must explicitly opt in.  This applies _in general_.

This particular text will be revisited with the resolution of #1520.